README.md

```diff
@@ -1,22 +1,22 @@
 # Secure Loader Project
 
-The secure bootloader allows the end-user of MAX78000 to load an encrypted firmware to MAX78000 based platforms and lock the SWD port. Once it is programmed, the firmware, including the CNN model and application code, cannot be extracted from the chip by anyone except the developer of the firmware who has the keys.
+The secure bootloader allows the end user of MAX78000 to load an encrypted firmware to MAX78000-based platforms and lock the SWD port. Once it is programmed, the firmware, including the CNN model and application code, cannot be extracted from the chip by anyone, except the developer of the firmware who has the keys.
 
 ## Programming the [MAX78000](https://www.analog.com/en/products/max78000.html) with the Bootloader Firmware
 
 _Note: Install the [Analog Devices MSDK](https://www.analog.com/en/design-center/evaluation-hardware-and-software/software/software-download?swpart=SFW0010820A) before following the steps below!_
 
-Use the following steps to program the [MAX78000FTHR](https://www.analog.com/en/design-center/evaluation-hardware-and-software/evaluation-boards-kits/max78000fthr.html) board with bootloader code. For complete list of documentation see [Documentation](#documentation) below.
+Use the following steps to program the [MAX78000FTHR](https://www.analog.com/en/design-center/evaluation-hardware-and-software/evaluation-boards-kits/max78000fthr.html) board with bootloader code. For a complete list of documentation, see [Documentation](#documentation) below.
```

Comment:
This brings up a good point that our procedure for flashing a specific binary file to our parts is not straightforward.  I think we can implement a feature in [flash.mk](https://github.com/analogdevicesinc/msdk/blob/main/Tools/Flash/flash.mk) to allow the user to specify a file.  i.e.

```shell
make flash.openocd BIN_FILE=MAX78000_Bootloader_vx_x_x.bin
```